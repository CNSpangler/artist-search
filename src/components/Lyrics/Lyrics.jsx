import React from 'react';
import PropTypes from 'prop-types';

const Lyrics = ({ artistName, songTitle, lyrics }) => (
  <>
    <h3>{songTitle} by {artistName}</h3>
    <p>{lyrics}</p>
  </>
);

Lyrics.propTypes = {
  artistName: PropTypes.string.isRequired,
  songTitle: PropTypes.string.isRequired,
  lyrics: PropTypes.string
};

export default Lyrics;
