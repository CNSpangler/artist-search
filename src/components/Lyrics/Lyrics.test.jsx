import React from 'react';
import { shallow } from 'enzyme';
import Lyrics from './Lyrics.jsx';

describe('Lyrics component', () => {
  it('renders Lyrics', () => {
    const wrapper = shallow(<Lyrics />);
    expect(wrapper).toMatchSnapshot();
  });
});
