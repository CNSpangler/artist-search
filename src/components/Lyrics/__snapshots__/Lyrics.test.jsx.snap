// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lyrics component renders Lyrics 1`] = `
<Fragment>
  <h3>
     by 
  </h3>
  <p />
</Fragment>
`;
